junction/static/bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "junction",
+  "version": "0.1.0",
+  "homepage": "https://github.com/pythonindia/junction",
+  "authors": [
+    "Mudassir <mudassir.dev@gmail.com>"
```

Comment:
Can we not have `authors` not duplicated here, apart from what we have in `CONTRIBUTORS.txt`? If i'm  not mistaken this is not a required property for a valid `package.json` 

Thanks.